packages/framework/src/Resources/views/Admin/Form/theme.html.twig

```diff
@@ -269,22 +269,32 @@
             {% if js_container is not null %}
                 <div class="{{ js_container.container_class }}" data-type="{{ js_container.data_type }}">
             {% endif %}
-            <div class="form-line">
+            <div class="form-line{% if multidomain is defined and isMultidomain() %} form-line--no-horizontal-gap{% endif %}">
```

Comment:
in the code above, there is a different condition (`if multidomain is defined and multidomain`), however, here, and in the other usages, there is just `if multidomain is defined and isMultidomain()` so the value of `multidomain` attribute has no impact on the condition evaluation, is that intended?